doc/enterprise/flight_recorder.rst

```diff
@@ -1,218 +1,74 @@
-    So, this value defines how much historical metrics data is collected up to the moment of crash.
-    The frequency of metric dumps is defined by :ref:`flightrec_metrics_interval <cfg_flightrec_metrics_interval>`.
-
-    | Type: integer
-    | Default: 180
-    | Environment variable: TT_FLIGHTREC_METRICS_PERIOD
-
-
-.. _cfg_flightrec_metrics_interval:
-
-.. confval:: flightrec_metrics_interval
-
-    Specifies the time interval (in seconds) that defines the frequency of dumping metrics.
-    This value shouldn't exceed :ref:`flightrec_metrics_period <cfg_flightrec_metrics_period>`.
-
-    | Type: number
-    | Default: 1.0
-    | Minimum: 0.001
-    | Environment variable: TT_FLIGHTREC_METRICS_INTERVAL
-
-.. NOTE::
-
-    Given that the average size of a metrics entry is 2 kB,
-    you can estimate the size of the metrics storage as follows:
-
-    .. code-block:: console
-
-        (flightrec_metrics_period / flightrec_metrics_interval) * 2 kB
-
-
-
-.. _config-requests:
-
-Requests
-~~~~~~~~
-
-This section lists the flight recorder settings related to
-storing the :ref:`request and response <internals-requests_responses>` data.
-
-* :ref:`flightrec_requests_size <cfg_flightrec_requests_size>`
-* :ref:`flightrec_requests_max_req_size <cfg_flightrec_requests_max_req_size>`
-* :ref:`flightrec_requests_max_res_size <cfg_flightrec_requests_max_res_size>`
-
-.. _cfg_flightrec_requests_size:
-
-.. confval:: flightrec_requests_size
-
-    Specifies the size (in bytes) of storage for the request and response data.
-    You can set this parameter to ``0`` to disable a storage of requests and responses.
-
-    | Type: integer
-    | Default: 10485760
-    | Environment variable: TT_FLIGHTREC_REQUESTS_SIZE
-
-
-
-.. _cfg_flightrec_requests_max_req_size:
-
-.. confval:: flightrec_requests_max_req_size
-
-    Specifies the maximum size (in bytes) of a request entry.
-    A request entry is truncated if this size is exceeded.
+Configuring a flight recorder
+-----------------------------
 
-    | Type: integer
-    | Default: 16384
-    | Environment variable: TT_FLIGHTREC_REQUESTS_MAX_REQ_SIZE
+When the flight recorder is enabled, you can set the options related to :ref:`logging <cfg_logging-log>`,
+:ref:`metrics <metrics-reference>`, and storing the :ref:`request and response <internals-requests_responses>` data.
 
+The ``flightrec`` configuration might look as follows:
 
-.. _cfg_flightrec_requests_max_res_size:
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/flightrec/config.yaml
+    :language: yaml
+    :start-at: flightrec:
+    :end-at: 10485780
+    :dedent:
 
-.. confval:: flightrec_requests_max_res_size
+In the example, the following options are set:
 
-    Specifies the maximum size (in bytes) of a response entry.
-    A response entry is truncated if this size is exceeded.
+*   :ref:`flightrec.logs_size <configuration_reference_flightrec_logs_size>` -- a log storage size in bytes
```

Comment:
Don't remember our guidelines for lists but dot at the end of long description might look better IMO :)